integration/script/run-test

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash -i
+
+function start_test_demo {
+  cd $curr/app/test-demo
+  uuid=$(uuidgen)
+  mkdir -p $curr/logs
+  echo "Starting the test demo"
+  npm run start 2>&1 | tee -a $curr/logs/$uuid.log &
+}
+
+function did_server_start {
+  if ! lsof -i:$1 > /dev/null; then
+    echo 0
+  else
+    echo 1
+  fi
+}
+
+function wait_for_demo_to_start {
+  echo "Waiting for test demo to start"
+  threshold=60
+  started=0
+  retry_count=1
+  while [ $started -eq 0 ] && [ $retry_count -lt $threshold ]
+  do
+    started=$(did_server_start $1)
+    sleep 5
+    retry_count=$(($retry_count+1))
+  done
+
+  if [ $retry_count -eq $threshold ]; then
+    echo "test demo not started"
+    exit 1
+  fi
+}
+
+echo `pwd`
+curr=`pwd`
+
+start_test_demo
+wait_for_demo_to_start 9000
+cd $curr
+
+# Start test and store result in /logs
+case $1 in
+  roster)
+    mocha test/roster_test.js > $curr/logs/$uuid-result 2>&1
```

Comment:
For understanding:
- How are these logs helpful to us? Are these just for the temporary runs on the VM or SauceLabs will capture these logs?
- Will adding the test-name help to the `$uuid-result` ($uuid-roster-test-result)?

Reply:
Good question.
The mocha runs the test one by one and while printing the test result, like how the unit test does.
But sometimes these test results mess up with the logs of API calls. Cause they share the same console.
For example:
It should be:
```
* test result
* test result
* test result
* test result
* test result
* test result
```

But it could be:
```
* test result
* test result
log of API calling
* test result
log of API calling
* test result
* test result
```

So I disabled this behavior and write the test result in a file to avoid this mess-up. 
At the end of the test, a) check the `passing` and `failing` string in the file, to check the whole testing result, b) print the whole test results in that file at the end of test for easy checking.

Reply:
Ok so this is just for checking whether a test failed or passed so should be fine it seems.

Reply:
Yes, the main goal is to check the test results and make it easier do it, similar to how we did in JS SDK integration test.

> Are these just for the temporary runs on the VM or SauceLabs will capture these logs?
It's the terminal log, so SauceLabs doesn't capture these logs. The GitHub runner will store them I think, but I don't know if we can access it.

> Will adding the test-name help to the $uuid-result ($uuid-roster-test-result)?
It helps just a little bit. Cause anyway you have to find the corresponding uuid to target the exact log. So I don't think there's a need to do it.

Reply:
Got it thanks!